packages/@webex/internal-plugin-metrics/src/metrics.js

```diff
@@ -103,6 +102,19 @@ const Metrics = WebexPlugin.extend({
     // is impossible so unable to use Date.now()
     payload.timestamp = new Date().valueOf();
 
+    return payload;
+  },
+
+  /**
+   * This corresponds to #sendSemiStructured() in the deprecated metrics handler
+   * @param {string} eventName
+   * @param {Object} props
+   * @param {string} preLoginId
+   * @returns {Object} HttpResponse object
+   */
+  submitClientMetrics(eventName, props = {}, preLoginId) {
```

Comment:
i would sync with @torgeadelin on this as he is making changes to the metrics file